nixos/modules/services/web-servers/caddy/default.nix

```diff
@@ -364,7 +364,7 @@ in
         # If the empty string is assigned to this option, the list of commands to start is reset, prior assignments of this option will have no effect.
         ExecStart = [ "" ''${cfg.package}/bin/caddy run ${runOptions} ${optionalString cfg.resume "--resume"}'' ];
         # Validating the configuration before applying it ensures we’ll get a proper error that will be reported when switching to the configuration
```

Comment:
It is a mistake to add this comment in https://github.com/NixOS/nixpkgs/pull/217536. See https://github.com/NixOS/nixpkgs/pull/217536#discussion_r1175279412.

While it is not relevant to this PR, it would be good to remove this comment.

Reply:
Yes, I considered pointing this out here as well (as part of my review).

But I eventually decided against doing so, because of an extremely fuzzy and far fetched interpretation of that comment:

> Validating the configuration before applying it ensures we’ll get a proper error that will be reported when switching to the configuration

being something like

> [Caddy] validat[es] the configuration before applying [, which] ensures we'll get a proper error that will be reported when switching to the configuration.

Caddy will never accept a broken config.

But yeah, that's really far fetched.

If you feel like updating the comment, please do :)

Reply:
Let's deal with this in a followup.